### PR TITLE
Make styles closer to original design

### PR DESCRIPTION
**Pull request by dasilvacontin**
Elements in general were bigger than they were meant to be. Some margins /
paddings were missing or incorrect.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
         </div>
         <div class="row top-description">
             <div class="col-md-12">
-                <p>Dream it. Hack it. Ship it. BarcelonaTech's first collegiate Hackathon. 400 hackers. 36h.</p>
+              <p>Dream it. Hack it. Ship it.<br>BarcelonaTech's first collegiate Hackathon.<br>400 hackers. 36h.</p>
```
Comment (mvdan) [inline dist/index.html]: indentation?

**Comment by mvdan** [comment]
Also, thanks for the thorough commit messages!

File: dist/index.html (hunk 1 above)
Comment (dasilvacontin) [inline dist/index.html]: Indentation is correct after my changes, it was wrong before. I didn't even notice, tbh.

**Comment by dasilvacontin** [comment]
@mvdan, no worries, you are welcome :)

File: dist/index.html (hunk 1 above)
Comment (mvdan) [inline dist/index.html]: I must be missing something. I thought we were doing 4 spaces?

Comment (dasilvacontin) [inline dist/index.html]: My bad, used to 2 spaces here.

Comment (dasilvacontin) [inline dist/index.html]: Fixed btw

**Pull request merged**